src/Pos/Client/Txp/History.hs

```diff
@@ -40,32 +40,32 @@ import           Data.Tagged                  (Tagged (..))
 import qualified Ether
 import           System.Wlog                  (WithLogger)
 
+import           Pos.Block.Core               (Block, mainBlockTxPayload)
 import           Pos.Constants                (blkSecurityParam)
 import           Pos.Context.Context          (GenesisUtxo (..))
+import           Pos.Core                     (Address, ChainDifficulty, HeaderHash,
+                                               difficultyL, prevBlockL)
 import           Pos.Crypto                   (WithHash (..), withHash)
 import           Pos.DB                       (MonadDB)
 import qualified Pos.DB.Block                 as DB
 import           Pos.DB.Error                 (DBError (..))
 import qualified Pos.DB.GState                as GS
 import           Pos.Slotting                 (MonadSlots)
 import           Pos.Ssc.Class                (SscHelpersClass)
-import           Pos.WorkMode.Class           (TxpExtra_TMP)
 #ifdef WITH_EXPLORER
-import           Pos.Explorer                 (eTxProcessTransaction)
+import           Pos.Explorer.Txp.Local       (eTxProcessTransaction)
```

Comment:
Wait, I don't think this change was in https://github.com/input-output-hk/cardano-sl/pull/304. Why was it introduced?